table/builder.go

```diff
@@ -307,67 +323,54 @@ func (b *Builder) Finish() error {
 	}
 	b.w.Reset(idxFile)
 
-	var encodeBuf [8]byte
-
 	// Don't compress the global ts, because it may be updated during ingest.
 	ts := uint64(math.MaxUint64)
 	if b.isExternal {
 		// External builder doesn't append ts to the keys, the output sst should has a non-MaxUint64 global ts.
 		ts = math.MaxUint64 - 1
 	}
-	binary.BigEndian.PutUint64(encodeBuf[:], ts)
-	if err := b.w.Append(encodeBuf[:]); err != nil {
+	if err := b.w.Append(u64ToBytes(ts)); err != nil {
```

Comment:
`ts` doesn't have an id?

Reply:
No, it is stored separately at the beginning of the index file. This field may be updated directly when doing ingest.

Reply:
Others are stored after `ts` and are compressed